.vscode/settings.json

```diff
@@ -1,4 +1,7 @@
 {
+  "editor.codeActionsOnSave": {
+    "source.organizeImports": true
```

Comment:
This change wouldn't have prevented the issue in this PR but sounds handy :)

Reply:
Maybe let's discuss this offline with the team 😄 Have been wanting to discuss removing this file, imo it's a bit overly opinionated to force `editor` level settings

Reply:
Sure, we can discuss, I think a little bit of structure is useful, so we don't end up with massive discrepancies in PRs due to contributor's different `tabSize` setting as an example.